README.md

```diff
@@ -86,10 +86,10 @@ For example:
 ### CIDv0
 
 CIDv0 is a backwards-compatible version, where:
-- the `multibase` is always `base58btc` and implicit (not written)
+- the `multibase` defaults to `base58btc` (if not written)
```

Comment:
👍 same in JS, although https://github.com/ipfs/go-ipfs/issues/4143 might become a thing soon

Reply:
ah wait, read this wrong. CIDv0 is always base58, there is no way to change that.